site/en/docs/extensions/reference/action/index.md

```diff
@@ -123,7 +123,8 @@ chrome.action.setBadgeBackgroundColor(
 
 An action's popup will be shown when the user clicks on the extension's button
```

Comment:
"extension's button" ... isn't this the action? We should perhaps call this UI element "action button" everywhere; WDYT?

Reply:
Edited to say the "extension's action button".